wasm/src/lib.rs

```diff
@@ -401,6 +401,6 @@ mod test {
             },
             1,
         );
-        assert!(result.is_ok());
+        assert!(result.is_err());
```

Comment:
Yes, I wanted to make sure it returned it to the previous behavior that treated it as a compilation error. I'm not sure exactly why this code asserted success before since you can see above that it also asserts an error is returned. In practice, the language service treats it as a compilation error and prevents the Run button from being active:
<img width="410" alt="image" src="https://github.com/microsoft/qsharp/assets/10567287/b5742388-61ab-4b97-a4fc-054f6c0d29a4">
Should I update this test case to call `compile` instead of `run_internal`?